ecs-cli/modules/cli/compose/entity/entity_test.go

```diff
@@ -120,6 +121,43 @@ func taskDefinition() *ecs.TaskDefinition {
 	}
 }
 
+// Test case ensures that GetContainersForTasksWithTaskNetworking() can be given
+// a list of tasks with task networking but without ENIs attached (i.e. task is provisioning) and return correctly.
+func TestGetContainersForTasksWithTaskNetworkingENIProvisioning(t *testing.T) {
+	ecsTasks := []*ecs.Task{
```

Comment:
can you add a comment here re: what this task is lacking? (i.e., "set up task without ENI/IP")

Reply:
I can, but I think the comment at the top of the test is pretty descriptive, its tasks with task networking, but without an ENIs (because the ENIs are still provisioning). I'm happy to edit that comment or add another one if you feel its needed, but I think that what I have there currently is sufficient and adding more would be redundant.

Reply:
I guess from looking at this code and not being super familiar with it, it wasn't immediately apparent to me where the lack of ENI was supposed to be determined (in the task struct or in some resource/call made with it later), so a comment here would help me understand where we're setting up the ENI-less-ness (?) we're testing against.